flexinput/src/main/java/com/lytefast/flexinput/model/Photo.kt

```diff
@@ -22,6 +26,16 @@ class Photo : Attachment<String> {
 
   constructor(parcelIn: Parcel) : super(parcelIn)
 
+  @TargetApi(Build.VERSION_CODES.Q)
```

Comment:
isn't there an annotation that's `RequiresApi`?

Reply:
yea thats better, forces static errors.